game-core/src/main/java/games/strategy/triplea/ui/mapdata/MapData.java

```diff
@@ -515,14 +515,18 @@ public Color impassableColor() {
   /**
    * Returns the polygons for any territories contained within a given territory.
    *
-   * @param territoryName Name of the territory, expected to be a sea territory, where we will be
-   *     checking for any contained 'island' territories.
+   * @param territoryName Name of the territory, where we will be checking for any contained
+   *     'island' territories.
+   * @return Empty if the parameter is not a sea territory or contains no 'islands', otherwise
+   *     returns any 'islands' contained within the given territory.
    */
   public Set<Polygon> getContainedTerritoryPolygons(final String territoryName) {
-    return contains.get(territoryName).stream()
```

Comment:
What about `contains.getOrDefault(territoryName, Set.of())`?

Reply:
I'm really on the fence. I like that it is more direct, but the later operations are what give me trouble. Specifically I don't like how it's kind of a 'hack' where a reader has to see "okay, in this case the set is empty, so we then get no-op, no-op, no-op, okay, we returned an empty set". Having it be more direct I think is more clear. IE: "if this does not contain, then we return empty set, otherwise we take the set, transform it, and return that".

Reply:
It is between:
```
    return !contains.containsKey(territoryName)
        ? Set.of()
        : contains.get(territoryName).stream()
            .map(this::getPolygons)
            .flatMap(Collection::stream)
            .collect(Collectors.toSet());
```

Or the getDefault with no-ops.

Reply:
Went with the 'getOrDefault' approach: 0edd2bc50

The `!contains` was also problematic as well.